.github/workflows/docs.yaml

```diff
@@ -0,0 +1,47 @@
+name: docs
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - '**/typedoc.js'
+      - '**/tsconfig.json'
+      - '**/package.json'
+      - 'modules/**'
```

Comment:
Since its just the JavaScript/TypeScript api documentation, can this be `modules/*.ts`?

thoughts? @trxcllnt @bryevdv @ajschmidt8

Reply:
Yeah a `modules/**/*.ts` should work!

Reply:
sounds good, i updated it to the following:

```yaml
paths:
      - '**/typedoc.js' # typedoc config changes
      - 'modules/**/src/**/*.ts' # src files, but not unit tests
      - '.github/workflows/**' # useful for ensuring that changes to the workflow file run successfully right after they're committed
```